### PR TITLE
[Swap the new index then merge this PR and deploy] Temp locations bibdata work 1198

### PR DESCRIPTION
**Pull request by christinach**
Should be merged and deployed to prod after the new index with the temp locations is in production.

### CODE REVIEW

**Comment by christinach** [comment]
Check record https://catalog-staging.princeton.edu/catalog/99124695833506421

**Comment by christinach** [comment]
Example records to review: 
https://catalog-staging.princeton.edu/catalog/99124695833506421
https://catalog-staging.princeton.edu/catalog/99118059693506421
https://catalog-staging.princeton.edu/catalog/99119415963506421
https://catalog-staging.princeton.edu/catalog/99106284373506421
https://catalog-staging.princeton.edu/catalog/99119428373506421
https://catalog-staging.princeton.edu/catalog/9939492373506421
https://catalog-staging.princeton.edu/catalog/99124695833506421
https://catalog-staging.princeton.edu/catalog/99122969053506421
https://catalog-staging.princeton.edu/catalog/9934199373506421
https://catalog-staging.princeton.edu/catalog/99115831133506421
https://catalog-staging.princeton.edu/catalog/99125385920806421 (this record I think is not configured correctly in alma, this is why it's showing lewis$res twice with items in holdings_1display but one is temporary and the other is permanent, even though in alma there are only 2 available items in the temp location.)

File: app/javascript/orangelight/availability.es6 [hunk 1]
```diff
@@ -133,10 +133,10 @@ export default class AvailabilityUpdater {
...
     for (let holding_id in holding_records) {
-      if (holding_id.match(/[a-zA-Z]\$[a-zA-Z]/) || holding_id.startsWith('fake_id_')) {
+      if (holding_id.match(/[a-zA-Z]\$[a-zA-Z]/)) {
         // In this case we cannot correlate the holding data from the availability API
         // (holding_records) with the holding data already on the page (from Solr).
-        // In this case we set all of them to "Check record" because we can get this
+        // In this case we set all of them to "View record for Full Availability" because we can get this
         // information in the Show page.
         const badges = $(`*[data-availability-record='true'][data-record-id='${record_id}'] span.availability-icon`);
         badges.text("View record for Full Availability")
```
Comment (christinach) [inline app/javascript/orangelight/availability.es6:142]: we would need to review the 'View record for full availability' after we swap the index with the temp locations.

**Comment by christinach** [comment]
We should merge and deploy after we swap the index with temp locations

File: app/javascript/orangelight/availability.es6 (hunk 1 above)
Comment (christinach) [reply app/javascript/orangelight/availability.es6:142]: https://github.com/pulibrary/orangelight/issues/2885

**Review by carolyncole — APPROVED** [review]
Looking good.  I might have kept the update to remove voyager and the temp update separate, but since they are so small it works!

**Pull request merged**